plugins/base.py

```diff
@@ -57,21 +57,18 @@ class BasePlugin(object):
     def __init__(self, settings):
         self.active = settings.get('active', False)
         self.debug = settings.get('debug', False)
+        self._logger = logging.getLogger(self.get_plugin_name())
 
-    def log_message(self, message, is_error=False):
-        msg = "[{} plugin] {}\n".format(self.get_plugin_name(), message)
-        if is_error:
-            sys.stderr.write(msg)
-        else:
-            sys.stdout.write(msg)
+    def log_message(self, message, level=logging.INFO):
```

Comment:
log_* methods are redundant. Use logger in each plugin directly. self.debug is not needed - logs level should be controlled when configuring the logger.